### PR TITLE
SAK-48373 - Roster: use a css class that actually exists

### PR DESCRIPTION
**Pull request by maurercw**
This fixes the NaN error, but there are still issues with the scroll-to-load content that I can't get figured out!

### CODE REVIEW

File: roster2/tool/src/webapp/js/roster.js [hunk 1]
```diff
@@ -848,7 +848,7 @@ roster.calculatePageSizes = function () {
 
   // height of container = height + top and bottom padding;
   // #roster-members-content has no height at load, so we approximate using the morpheus page container
-  var containerHeight = parseInt($('div.Mrphs-pagebody').height());
+  var containerHeight = parseInt($('div.portal-main-container').height());
```
Comment (adrianfish) [inline roster2/tool/src/webapp/js/roster.js:851]:
```suggestion
  const containerHeight = parseInt($('div.portal-main-container').height());
```

**Review by adrianfish — APPROVED** [review]
Small change suggestion. Was this all it took to get the infinite scroll working again?

**Comment by maurercw** [comment]
@adrianfish As mentioned in the earlier comment, this only fixes the NaN error.  Scroll-to-load is still not working.

**Comment by adrianfish** [comment]
@maurercw Thanks Chris. I'll look at the infinite scroll stuff for roster. Thanks for having a crack at it :)

**Pull request merged**